changelog.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## NEXT
+
+- Fixed a bug that would allow the addition of b--a and b->a edges to a graph with an existing undirected a--b edge in
+  the `cai_causal_graph.causal_graph.CausalGraph._set_edge` method.
+  - Added the `CausalGraphsError.ReverseEdgeExistsError` exception, which distinguishes errors arising from the
+    introduction of cycles or reverse edges.
+  - `TestCausalGraphPrinting.test_add_edges_from_single_path` and
```

Comment:
No need to put anything specific about tests in the changelog, especially for a public repo. Would probably just remove this bullet point